### DISCUSSION TITLE
This user account is blocked. Contact registration@vercel.com for more information.

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#2349] [Help] [answered]
### Summary

i cannot add my github account in vercel, vercel shows me "This user account is blocked. Contact registration@vercel.com for more information." this why?

### Additional information

no additional information

**Comment by 4ndual** [comment]
Same happened to me one year ago, the only answer i got so far was:

"Your account was blocked for hosting content that are against our TOS. Unfortunately, your account cannot be enabled."

not further explanation

**Comment by amyegan** [comment, marked as answer]
If you have not already, please email [registration@vercel.com](mailto:registration@vercel.com) with your username and a quick description of the project you are deploying. This discussion is public, so email is a better option to keep your account info private
